.github/actions/parse-version/index.js

```diff
@@ -0,0 +1,19 @@
+const {setOutput} = require('@actions/core');
+const headlessPackage = require('../../../packages/headless/package.json');
+const atomicPackage = require('../../../packages/atomic/package.json');
+
+/**
+ * @param {string} fullVersion
+ */
+function getVersions(fullVersion) {
+  const [minor, major] = /^([^\.]*)\.[^\.]*/.exec(fullVersion);
+  return {minor, major};
+}
```

Comment:
The split and the alternative regexp would not give the same result as the current solution, since `minor` in my example contains a period (e.g., `2.19`) and that's what we need.

I'll add `semver`.